### DISCUSSION TITLE
[Feature Request] Order files in PRs so that test files come first

### DISCUSSION THREAD

**Discussion by OldhamMade** [community/community#41806] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Some smaller studies have shown that reviewers do a better job with PRs when the test files (unit/integration/etc) are reviewed before the implementation files. It would be great if there was a per-project toggle/option to order the test files _before_ the implementation files, so they are at the top of PRs and will be reviewed first. This could be done with a pattern, so that GH doesn't need to _know_ about the files, it simply applies the pattern to the file paths and orders those before the rest of the files. This could be quite generic, allowing some customers to order documentation, or config, or XML files at the top of PRs for their org. It could even be done by the developer raising the PR (falling back to the project's setting).

**Comment by WebMamba** [comment]
I like to have this feature too. I work on a project following Behavior-driven development principles. So It could be great if my scenario appear first. For now the only way to order files it's to play with the name of the files.
